operations/rates.md

```diff
@@ -49,222 +52,158 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
-| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the Enterprises. If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `Extent` | [Rate Extent](rates.md#rate-extent) | required | Extent of data to be returned. |
+| `RateIds` | array of string | optional, max 1000 items | Unique identifiers of the requested [Rates](rates.md#rate). |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months |  |
 | `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of the [Services](services.md#service) from which the rates are requested. |
-| `RateIds` | array of [Rates](#rate) | optional, max 1000 items | Unique identifiers of the requested [Rates](rates.md#rate). |
-| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Rates](rates.md#rates) were updated. |
-| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of [Rate](#rate) from external systems. |
-| `ActivityStates` | array of string [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted, or both types of record. If not specified, both active and deleted will be returned. |
-| `Extent` | [Rate extent](#rate-extent) | required | Extent of data to be returned. |
+| `ExternalIdentifiers` | array of string | optional, max 1000 items | Identifiers of [Rate](rates.md#rate) from external systems. |
+| `ActivityStates` | array of [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted, or both types of record. If not specified, both active and deleted will be returned. |
 | `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
 
-#### Rate extent
+#### Rate Extent
+Extent of data to be returned.
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `Rates` | bool | optional | Whether the response should contain rates. |
-| ~~`RateGroups`~~ | ~~bool~~ | ~~optional~~ | ~~Whether the response should contain rate groups.~~ **Deprecated!** |
-| `AvailabilityBlockAssignments` | bool | optional | Whether the response should contain availability block assignments. |
+| `Rates` | boolean | required | Whether the response should contain rates. |
+| ~~`RateGroups`~~ | ~~boolean~~ | ~~required~~ | ~~Whether the response should contain rate groups.~~ **Deprecated!** Use `rateGroups/getAll`|
+| ~~`RateRestrictions`~~ | ~~boolean~~ | ~~required~~ | ~~Whether the response should contain rate restrictions.~~ **Deprecated!** Use `restrictions/getAll`|
```

Comment:
FYI: Changed from `optional` to `reqired` since all of them are booleans

Reply:
Yes it would be nice to have some clarity. Is the `Extent` JSON object required or optional? If optional, we should add to the description what the default behaviour is, e.g. "defaults to returning Rate only". The same goes for the fields within `Extent`, e.g. Rates optional "defaults to true".

Reply:
fixed [here](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/bc5e60d30b2030ce1134fd4e1a99861786d7159a)